packages/components/src/components/contract-card/contract-card-items/contract-card-body.tsx

```diff
@@ -143,6 +146,7 @@ const ContractCardBody = ({
                 contract_info={contract_info}
                 currency={currency}
                 getCardLabels={getCardLabels}
+                // @ts-expect-error Because component is jsx error is gone after ts-migration
```

Comment:
@henry-deriv will circle-ci or vercel checks fail without this comment? if not, it's ok to remove the comment and let the error be because this error is related to this component being jsx.🙂

Reply:
yes. i added this because tests were failing